python/src/DistFunc_doc.i.in

```diff
@@ -184,20 +183,22 @@ Examples
 // ---------------------------------------------------------------------
 
 %feature("docstring") OT::DistFunc::pHypergeometric
-"The cumulative probability function of an hypergeometric distribution.
+"Cumulative density function of an hypergeometric distribution.
```

Comment:
According to Wikipedia's [Cumulative_distribution_function](https://en.wikipedia.org/wiki/Cumulative_distribution_function), the correct name is neither cumulative probability function, nor cumulative density function, but cumulative distribution function.
```suggestion
"Cumulative distribution function of an hypergeometric distribution.
```

Reply:
Yes. I modified all the files: 

- CDF: cumulative distribution function
- PDF for a continuous distribution : probability density function
- PDF for a discrete distribution : probability distribution function